src/test/objectstore/store_test.cc

```diff
@@ -6855,6 +6855,90 @@ TEST_P(StoreTestSpecificAUSize, fsckOnUnalignedDevice2) {
   g_conf->apply_changes(NULL);
 }
 
+TEST_P(StoreTest, mergeRegionTest) {
```

Comment:
This test case (as well as above fsckOnUnaligned* which is currently missing) should be wrapped with #if defined(HAVE_LIBAIO).
That was the predecessor for (WITH_BLUESTORE) macros.

Reply:
Thanks for the hint. I did it, adjusted the conflicts description of the commit and force pushed the new commit.